src/inmanta/agent/agent_new.py

```diff
@@ -132,8 +134,18 @@ def periodic_schedule(
                 return True
             return False
 
-        periodic_schedule("deploy", self.scheduler.deploy, self._deploy_interval, self._deploy_splay_value)
-        periodic_schedule("repair", self.scheduler.repair, self._repair_interval, self._repair_splay_value)
+        periodic_schedule(
+            "deploy",
+            functools.partial(self.scheduler.deploy, TaskPriority.INTERVAL_DEPLOY).func,
```

Comment:
I used `func` because of this line that is called further in the code:
`LOGGER.info("Calling %s", action.__name__)`

and `functools.partial` does not have the `__name__` attr.  So this solution does not work.
I'll just go back to having dedicated function for this